themes/bootstrap3/templates/channels/channelList.phtml

```diff
@@ -26,8 +26,8 @@
 <?php if (empty($token)): ?>
   <form action="<?=$this->url('channels-search')?>" class="channel-search form-inline">
     <?=$this->transEsc('channel_searchbox_label')?>
-    <input type="text" name="lookfor" class="form-control" value="<?=$this->escapeHtmlAttr($this->lookfor) ?>">
-    <input type="submit" value="<?=$this->transEscAttr('Submit')?>" class="btn btn-default">
+    <input type="text" name="lookfor" class="form-control" value="<?=$this->escapeHtmlAttr($this->lookfor) ?>" aria-label="<?=$this->transEscAttr('search_terms') ?>">
+    <input type="submit" value="<?=$this->transEscAttr('Submit')?>" class="btn btn-default" aria-label="<?=$this->transEscAttr('Submit') ?>">
```

Comment:
This label is redundant since `<input type="submit">` elements already [use their value attribute for screen readers](https://russmaxdesign.github.io/accessible-forms/01-input-submit.html).

Reply:
There's a few more `type="submit"` and `type="button"` in the myresearch forms farther down.